### DISCUSSION TITLE
Marketplace for benthos recipes

### DISCUSSION THREAD

**Discussion by mannharleen** [redpanda-data/connect#977] [Ideas]
Core idea: While the core product is managed and maintained by the core maintainer(s), the wider "user community" of an OSS project can contribute (back) by [providing feedback (of course!)] sharing the recipes that they have built and are using. These recipes can be "config" files, "templates" or "resources?".

Questions to ponder upon:
- How will users submit/update/delete recipes?
- How will we control the quality?
- How do we limit duplication?
- What information is to be captured along with the recipe? e.g. author, repo??
- Others..?

**Comment by Jeffail** [comment]
The ideal end goal would be a web application made specifically for this purpose, but if anyone knows of an existing space or app that might suit this kind of thing I'd love to check them out.

**Reply by mannharleen** [reply]
> The ideal end goal would be a web application made specifically for this purpose, but if anyone knows of an existing space or app that might suit this kind of thing I'd love to check them out.

Here is one https://flows.nodered.org/
With NR, though I take ideas from the marketplace, I don't actually use them the recipes directly in my "flows". And the reason is simeple - lack of quality in there.
